builtin/locale/__builtin.ru.tr

```diff
@@ -0,0 +1,246 @@
+<name> (<privilege> [, <privilege2> [<...>]] | all)=<имя> (<привилегия> [, <привилегия2> [<...>]] | all)
+Give privileges to player=Предоставить привилегии игроку
+Invalid parameters (see /help grant).=Недопустимые параметры (см. /help grant).
+<privilege> [, <privilege2> [<...>]] | all=<привилегия> [, <привилегия2> [<...>]] | all
+Grant privileges to yourself=Предоставить привилегии себе
+Invalid parameters (see /help grantme).=Недопустимые параметры (см. /help grantme).
+Your privileges are insufficient. '@1' only allows you to revoke: @2=Ваших привилегий недостаточно. '@1' позволяет вам отозвать только: @2
+Note: Cannot revoke in singleplayer: @1=Примечание: Невозможно отозвать в одиночной игре: @1
+Note: Cannot revoke from admin: @1=Примечание: Невозможно отозвать у администратора: @1
+No privileges were revoked.=Никакие привилегии не были отозваны.
+@1 revoked privileges from you: @2=@1 отозвал у вас привилегии: @2
+Remove privileges from player=Снять привилегии с игрока
+Invalid parameters (see /help revoke).=Недопустимые параметры (см. /help revoke).
+Revoke privileges from yourself=Отозвать привилегии у себя
+Invalid parameters (see /help revokeme).=Недопустимые параметры (см. /help revokeme).
+<name> <password>=<имя> <пароль>
+Set player's password (sent unencrypted, thus insecure)=Установить пароль игрока (отправляет в незашифрованном виде, следовательно, небезопасный)
+Name field required.=Обязательное поле "Имя".
+Your password was cleared by @1.=@1 сбросил ваш пароль.
+Password of player "@1" cleared.=Пароль игрока "@1" сброшен.
+Your password was set by @1.=@1 установил вам пароль.
+Password of player "@1" set.=Установлен пароль игрока "@1".
+<name>=<имя>
+Set empty password for a player=Установить пустой пароль для игрока
+Reload authentication data=Перезагрузить аутентификационные данные
+Done.=Готово.
+Failed.=Не удалось.
+Remove a player's data=Удалить данные игрока
+Player "@1" removed.=Игрок "@1" удален.
+No such player "@1" to remove.=Нет такого игрока "@1", которого можно было бы удалить.
+Player "@1" is connected, cannot remove.=Игрок "@1" сейчас в игре, удалить его невозможно.
+Unhandled remove_player return code @1.=Необработанный код возврата remove_player @1.
+Cannot teleport out of map bounds!=Невозможно телепортироваться за пределы карты!
+Cannot get player with name @1.=Не удается получить игрока с именем @1.
+Cannot teleport, @1 is attached to an object!=Невозможно телепортироваться, @1 привязан к объекту!
+Teleporting @1 to @2.=Телепортация @1 в @2.
+One does not teleport to oneself.=Игрок не может телепортироваться к самому себе.
+Cannot get teleportee with name @1.=Не удается найти телепортирующегося с именем @1.
+Cannot get target player with name @1.=Не удается найти целевого игрока с именем @1.
+Teleporting @1 to @2 at @3.=Телепортация @1 к @2 в @3.
+<X>,<Y>,<Z> | <to_name> | <name> <X>,<Y>,<Z> | <name> <to_name>=<X>,<Y>,<Z> | <to_name> | <имя> <X>,<Y>,<Z> | <имя> <имя_к_кому>
```

Comment:
You may translate `<to_name>` (in case you missed it).

Reply:
```suggestion
<X>,<Y>,<Z> | <to_name> | <name> <X>,<Y>,<Z> | <name> <to_name>=<X>,<Y>,<Z> | <имя_к_кому> | <имя> <X>,<Y>,<Z> | <имя> <имя_к_кому>
```